utils/redirects.ts

```
@@ -0,0 +1,135 @@
+   }
+ }
+ return files;
+}
+
+function extractLinks(content: string): string[] {
+ const markdownLinkRegex = /\[([^\]]+)\]\(([^)]+)\)/g;
+ const hrefRegex = /href="([^"]+)"/g;
+ const links: string[] = [];
+ 
+ let match;
+ while ((match = markdownLinkRegex.exec(content)) !== null) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)